rust/lib/Cargo.toml

```diff
@@ -8,7 +8,7 @@ edition = "2021"
 
 [dependencies]
 jni = { version = "0.10.2", default-features = false }
-zecwalletlitelib = { git="https://github.com/zingolabs/zecwalletlitelib", default-features=false, rev="3b2ef3a72df6f740bf29a4a86db2624a313c0766"}
+zecwalletlitelib = { git="https://github.com/zingolabs/zecwalletlitelib", default-features=false, rev="d50ce9d618c1a2adc478ffbc3ad281512fbb0c15" }
```

Comment:
Let's make this human readable by using `tag = "vzing-0.0.1"`.

Git tags, are not (supposed to be) moveable (like branches).